lib/game.rb

```diff
@@ -0,0 +1,92 @@
+class Game
+  @@v_conditions = [[1,2,3], [4,5,6], [7,8,9], [1,4,7], [2,5,8], [3,6,9], [1,5,9], [3,5,7]]
```

Comment:
I suppose this is the winning combination. 
1. You can name it more intuitively (like winning_conditions or victory_conditions)
2. You don't need it to be a Class variable. An instance variable is ok